src/Features/Core/Portable/AddImport/SymbolReferenceFinder_PackageAssemblySearch.cs

```diff
@@ -121,7 +121,7 @@ private partial class SymbolReferenceFinder
             {
                 cancellationToken.ThrowIfCancellationRequested();
                 var results = await _symbolSearchService.FindPackagesWithTypeAsync(
-                    source.Name, name, arity, cancellationToken).ConfigureAwait(false);
+                    source.Source, name, arity, cancellationToken).ConfigureAwait(false);
```

Comment:
does this actually fix the issue? I think you need to completely remove the usage of name. for example, https://github.com/dotnet/roslyn/pull/35397#discussion_r297088075 probably will still have issue?

Reply:
> does this actually fix the issue?

Yep this fixes it

![image](https://user-images.githubusercontent.com/9797472/60207272-52566780-980a-11e9-88a7-ad8cbd701177.png)